network-api/networkapi/settings.py

```diff
@@ -85,6 +88,18 @@
 else:
     environ.Env.read_env()
 
+SENTRY_DSN = env('SENTRY_DSN')
+HEROKU_RELEASE_VERSION = env('HEROKU_RELEASE_VERSION')
+
+if SENTRY_DSN:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration()],
+        release=HEROKU_RELEASE_VERSION
```

Comment:
We'll need to enable the Dyno Metadata feature on staging and production to make this variable available to use: https://devcenter.heroku.com/articles/dyno-metadata

Reply:
Done for both staging and prod.